climada/engine/cost_benefit.py

```diff
@@ -134,28 +136,40 @@ class CostBenefit():
         'impact' (Impact): impact instance
     """
 
-    def __init__(self):
+    def __init__(self,
+                 present_year: Optional[int] = DEF_PRESENT_YEAR,
+                 future_year: Optional[int] = DEF_FUTURE_YEAR,
+                 tot_climate_risk: Optional[float] = 0.0,
+                 units:  Optional[str] = 'USD',
+                 color_rgb: Optional[dict] = None,
+                 benefit: Optional[dict] = None,
+                 cost_ben_ratio: Optional[dict] = None,
+                 imp_meas_present: Optional[dict] = None,
+                 imp_meas_future: Optional[dict] = None
```

Comment:
Possibly specify data types of dict keys and values using `typing.Dict`?

Reply:
I'm not sure how to do it for the imp_meas_present and imp_meas_future which are quite complex dictionaries